modules/RTC/TPCUtils.js

```diff
@@ -420,16 +449,31 @@ export class TPCUtils {
     }
 
     /**
-    * Enables/disables audio transmission on the peer connection. When
-    * disabled the audio transceiver direction will be set to 'inactive'
-    * which means that no data will be sent nor accepted, but
-    * the connection should be kept alive.
-    * @param {boolean} active - true to enable audio media transmission or
-    * false to disable.
-    * @returns {void}
-    */
-    setAudioTransferActive(active) {
-        this.setMediaTransferActive(MediaType.AUDIO, active);
+     * Resumes or suspends media on the peerconnection by setting the active state on RTCRtpEncodingParameters
+     * associates with all the senders that have a track attached to it.
+     *
+     * @param {boolean} enable - whether media needs to be enabled or suspended.
+     * @returns {Promise} - A promise that is resolved with an array of results when the change is succesful on all the
+     * senders, rejected otherwise.
+     */
+    resumeOrSuspendMedia(enable) {
```

Comment:
We already have `setMediaTransferActive`, didn't want to confuse it with that one. I could rename the other one to `setVideoTransferActive` since its being used for exclusively for turning off incoming and outgoing video for p2p for lastn=0 and name this `setMediaTransferActive`. Does that work ?

Reply:
Sounds good! How about going a bit further and call the P2P one setP2pVideoTransferActive ?